bin/pycbc_optimize_snr

```diff
@@ -19,11 +19,17 @@ from pycbc.conversions import (
     mass1_from_mtotal_eta, mass2_from_mtotal_eta
 )
 from scipy.optimize import differential_evolution, shgo
+import pyswarms as ps
 from pycbc.io import live
 from pycbc.io.hdf import load_hdf5_to_dict
 from pycbc.detector import Detector
 from pycbc.psd import interpolate
 
+try: 
+    import pyswarms as ps
+except:
+    print("No pyswarms package available for pso optimization")
```

Comment:
I suggest setting `ps = None` instead of printing a warning here. Later, if the user selects `--optimizer pso`, you can check if `ps is None` and if so, then use `parser.error()` to print an error message. No need to print a warning if the user does not want PSO in the first place.

Reply:
I have added the error message and it works, I just want to make sure I have used the right formatting for it